sdk/storage/Azure.Storage.Queues/src/QueueMessageEncoding.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Queues
+{
+    /// <summary>
+    /// TODO (kasobol-msft) add docs.
+    /// </summary>
+    public enum QueueMessageEncoding
+    {
+        /// <summary>
+        /// TODO (kasobol-msft) add docs.
+        /// </summary>
+        UTF8 = 0,
```

Comment:
Sound we add `None = 0` as a default?

Reply:
I had `None = 0` before as a default, which meant - treat bytes as UTF8. There was suggestion in discussion above that `None = 0` doesn't say what happens so it got renamed to `UTF8` and became a default.